### PR TITLE
Sign language into basic translation gloves

### PR DESCRIPTION
**Pull request by LeiteQualhado**
Adds gloves that can translate sign language into basic and works as a radio to the loadout

### Asset Licenses
I did not create anything here, i'm just making a PR in behalf of UponASeaOfStars

### CODE REVIEW

**Comment by LeiteQualhado** [comment]
!review required

**Comment by LeiteQualhado** [comment]
> Personally this seems like something that should be R&D. Besides that it should probably be more than just 1 loadout point if being a part of the loadout selection.

If maintainers think the point cost should be higher i can send a message to UponASeaOfStars and ask for a point increase
But i disagree it should be a RnD thing, it's an accessibility item besides we don't always have science staff playing

File: code/game/objects/items/devices/radio/signgloves.dm [hunk 1]
```diff
@@ -0,0 +1,36 @@
+/obj/item/device/radio/gloves
```
Comment (alsoandanswer) [inline code/game/objects/items/devices/radio/signgloves.dm:1]: This file needs to be ticked

**Comment by alsoandanswer** [comment]
Should be discussed with CCIAA team on how this works with Job Accessibility

**Comment by UponASeaOfStars** [comment]
The reason it's in the loadout rather than R&D is because it's an accessibility item, and it'd be kinda shitty to have it be only available _sometimes_ for characters that rely on it to communicate.

**Comment by NonQueueingMatt** [comment]
This is fine, but characters using these will still be counted as mute for the purpose of accessibility guidelines.

**Comment by Sniblet** [comment]
why is this closed

**Comment by Sniblet** [comment]
oh because a ban happened and so the same code needs to be prd again by a different author even though the current author wasnt banned either
i see
ill do it

**Pull request closed without merging**